contracts/JBProjects.sol

```diff
@@ -118,20 +127,29 @@ contract JBProjects is JBOperatable, ERC721Votes, Ownable, IJBProjects {
     function setMetadataOf(uint256 _projectId, JBProjectMetadata calldata _metadata)
         external
         override
-        requirePermission(ownerOf(_projectId), _projectId, JBOperations.SET_PROJECT_METADATA)
+        requirePermissionAllowingOverride(
```

Comment:
should we just make these available through the controller through onlyController()?

Reply:
Good call, im down, do you want to do only `setMetadataOf` or should I do it for the others as well?

Reply:
others as well

Reply:
should we do this in `JBDirectory` too? that'd remove the dependence of the directory on fcstore, but if a controller doesn't allow changing a controller, that'll never be allowed. 

could be good.

Reply:
ooo. i also hear you about keeping jbdirectory (immutable contract) further from some risk potential

Reply:
incldue JBTokenStore.transferFor